fop/fom.c

```diff
@@ -638,17 +638,14 @@ M0_INTERNAL void m0_fom_queue(struct m0_fom *fom)
 	fom->fo_loc = dom->fd_localities[loc_idx];
 	fom->fo_loc_idx = loc_idx;
 	m0_fom_sm_init(fom);
-	addb2_introduce(fom);
-	m0_fom_locality_inc(fom);
-	fom->fo_cb.fc_ast.sa_cb = &queueit;
-	if (m0_reqh_service_state_get(fom->fo_service) == M0_RST_STOPPED ||
-		m0_reqh_state_get(m0_fom_reqh(fom)) == M0_REQH_ST_STOPPED) {
+	m0_atomic64_inc(&fom->fo_service->rs_fom_queued);
+	if (m0_reqh_service_state_get(fom->fo_service) == M0_RST_STOPPED) {
```

Comment:
Set fom rc to something.

Reply:
@nikitadanilov : You mean m0_fom_queue() should return error like -ESHUTDOWN, and then caller of this function needs to handle this error. If so, currently return type of m0_fom_queue() is void, which needs to be changed and all the callers too needs to be updated.
For now i have added the log message. 

What do you suggest here ?

Reply:
Log is added here.